BigBaseV2/src/views/vehicle/view_vehicle.cpp

```diff
@@ -58,24 +58,31 @@ namespace big
 		components::small_text("Auto Drive");
 
 		components::button("Drive To Waypoint", [] {
+
 			g->vehicle.auto_drive_to_waypoint = true;
-		});
+			});
 
 		components::button("Wander", [] {
+
 			g->vehicle.auto_drive_wander = true;
-		});
+			});
 
 		ImGui::SliderInt("Top Speed", &g->vehicle.auto_drive_speed, 1, 200);
 
 		components::button("E-Stop", [] {
-			g->vehicle.auto_drive_to_waypoint = false;
-			g->vehicle.auto_drive_wander = false;
-			VEHICLE::SET_VEHICLE_FORWARD_SPEED(self::veh, 0);
-			TASK::CLEAR_VEHICLE_TASKS_(self::veh);
-			TASK::CLEAR_PED_TASKS(self::ped);
-		});
-
-		if (ImGui::ListBox("Driving Style", &g->vehicle.driving_style_id, vehicle::driving_style_names, 3))
+
+			QUEUE_JOB_BEGIN_CLAUSE()
+			{
+				g->vehicle.auto_drive_to_waypoint = false;
+				g->vehicle.auto_drive_wander = false;
+				VEHICLE::SET_VEHICLE_FORWARD_SPEED(self::veh, 0);
+				TASK::CLEAR_VEHICLE_TASKS_(self::veh);
+				TASK::CLEAR_PED_TASKS(self::ped);
+			}
+			QUEUE_JOB_END_CLAUSE
```

Comment:
Wrapped inside a queue job? The button component callback automatically wraps its callbacks in fiber jobs, no need for an additional queue job.